### DISCUSSION TITLE
Community version of Swarm learning - v1.2.0

### DISCUSSION THREAD

**Discussion by suresh-ls** [HewlettPackard/swarm-learning#147] [Announcements]
We’re excited to announce Swarm Learning 1.2.0 community release!

This release contains new features and important bug fixes.
 1. Reverse Proxy – Feature to reduce number of ports to be opened for Swarm training
 2. AMD GPU – Feature to enable work load training on AMD GPUs 
 3. Provision to opt out of Leader election for any SL node. If user doesn’t want to make a slow node (with less compute power, network band width etc) as merge leader, then ‘SL_MAKE_ME_ADMIN’ can be set to ‘False’.
 4. SL-UID mapping in GET TASKRUNNER PEER STATUS command for better debuggability
 5. Improved List nodes command to display the list of Swarm nodes that have registered and are currently active
 6. SWCI aborts if the task fails in between ‘exit on error’ command
 7. Corrected SWCI error reporting
 8. Corrected run-swci script to handle swci_init file
 9. Version agnostic container info to run SWOP container across Linux distributions
10. Improved SWCI inline HELP
11. Improved plotTopology() in SWCI web API

Thank you all for your support! Please let us know for any feedback or queries.

**Comment by IMOKURI** [comment]
The following directory has only README, but could you please publish the example configuration file?

https://github.com/HewlettPackard/swarm-learning/tree/master/examples/reverse-proxy/mnist
